test/e2e/suite-features/external-sd.go

```diff
@@ -23,10 +23,10 @@ import (
 	"os"
 	"time"
 
+	"github.com/apache/apisix-ingress-controller/pkg/config"
 	"github.com/onsi/ginkgo/v2"
 	"github.com/stretchr/testify/assert"
 
-	"github.com/apache/apisix-ingress-controller/pkg/config"
```

Comment:
this one don't need change.

Reply:
Goland IDE formatted it automatically. 😂